app/views/invoices/show.html.erb

```diff
@@ -1 +1,22 @@
-<%= @invoice.status %>
+<section>
+  <h3>Invoice: </h3>
+  <%= @invoice.id %>
+  <%= @invoice.status %>
+  <%= @invoice.created_at.strftime('%A, %b %d %Y') %>
+</section>
+
+<section>
+<h3>Customer: <%= @invoice.customer.last_name %>, <%= @invoice.customer.first_name %></h3>
```

Comment:
Thinking of making a method to concat the first and last name, but can regroup and decide where to place that! But looks good